### DISCUSSION TITLE
Swan Chain, what do you think you will offer us?

### DISCUSSION THREAD

**Discussion by bildevs** [swanchain/governance#3] [General]
Swan Chain, what do you think you will offer us?

**Comment by flyworker** [comment]
You can refer to https://docs.swanchain.io/core-concepts/readme about the core value providing, but welcome for more questions

**Comment by gugege521** [comment]
据我所了解的，swan chain应该是一个去中心的云服务商，可以类比亚马逊云来理解，亚马逊云能够提供的，swan chain将能够以更低的价格和更可靠的数据传输和安全来提供

**Comment by daiyiheng001** [comment]
Calling all community members: Shape the future of Swan Chain!
Propose, discuss, vote - make your voice heard.
Join us in pioneering blockchain governance innovation!
